pkg/kubelet/kubelet.go

```diff
@@ -3014,7 +3014,7 @@ func (kl *Kubelet) RunInContainer(podFullName string, podUID types.UID, containe
 
 	var buffer bytes.Buffer
 	output := ioutils.WriteCloserWrapper(&buffer)
-	err = kl.runner.ExecInContainer(container.ID, cmd, nil, output, output, false, nil)
+	err = kl.runner.ExecInContainer(container.ID, cmd, nil, output, output, false, nil, 0)
```

Comment:
This PR is not changing `RunInContainer`, therefore this is calling `kl.runner.ExecInContainer` without setting a timeout. I didn't look at what uses `RunInContainer`, but perhaps it would benefit from timeouts (and the resize channel?).